core/frontend/src/tools/ViewTool.ts

```diff
@@ -2128,7 +2128,7 @@ class ViewLookAndMove extends ViewNavigate {
         if (pixel.distanceFraction < 0)
           continue; // No geometry at location...
 
-        const hitPointWorld = vp.getPixelDataWorldPoint({pixels, x: testPoint.x, y: testPoint.y, preserveModelDisplayTransforms: true});
+        const hitPointWorld = vp.getPixelDataWorldPoint({ pixels, x: testPoint.x, y: testPoint.y, preserveModelDisplayTransforms: true });
```

Comment:
VS Code made this change automatically. I'm not sure if that is due to my global settings or due to some workspace settings that weren't present the last time this file was edited. @calebmshafer should I leave this in, or do another commit that undoes this change?